changes/changelog.md

```diff
@@ -28,78 +29,77 @@ The following rules have been removed as this is consumer-facing content and alr
 ### Update
 
 - Profile and link relation URLs can be independent of the API Portal.
-- Renamed the rule "MUST use absolute URLs for custom link relation types" to ["MUST use absolute URIs for custom link relation types"](/guidelines/r100037). The requirement for a `https://api.otto.de/portal` prefix has been removed, and it is no longer required to provide resolvable link relation URIs.
-- Merged the "Profile" section previously being found in "Hypermedia" into the section ["Versioning of incompatible changes"](/guidelines/rest-guidelines/compatibility#versioning-of-incompatible-changes), because profiles are used for versioning of hypermedia and non-hypermedia APIs.
+- Renamed the rule "MUST use absolute URLs for custom link relation types" to "MUST use absolute URIs for custom link relation types [R100037](/guidelines/r100037)". The requirement for a `https://api.otto.de/portal` prefix has been removed, and it is no longer required to provide resolvable link relation URIs.
```

Comment:
Unfortunately absolute URLs are not working. We need to use relative URLs (starting with ./ or ../) or use the API Potal Links (https://api.otto.de/portal/guidelines/r100037)